lib/svg-sprite.js

```diff
@@ -28,6 +27,21 @@ const Layouter = importLazy(() => require('./svg-sprite/layouter.js'))();
 const svgo = importLazy(() => require('./svg-sprite/transform/svgo.js'))();
 const ArgumentError = importLazy(() => require('./svg-sprite/errors/argument-error.js'))();
 
+/**
+ * Trim the start of a string of the specified characters.
+ *
+ * @param {string} name The string to trim the start of.
+ * @param {string[]} [charsToTrim=[]] The characters to trim from the start of the string.
+ * @returns {string} The trimmed string.
+ */
+function trimStart(name, charsToTrim = []) {
+  const nameArray = Array.from(name);
```

Comment:
1. Please stick to an existing API, the second argument should be a string (the default value is a space character) 
2. String already have [indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf) method

Reply:
`String.indexOf` can only search for a fixed substring, this function requires searching for the first character in a string that is not in the unordered list of characters to remove.

Reply:
Got it.